consent-ws/src/main/java/org/broadinstitute/consent/http/service/DatabaseSummaryAPI.java

```diff
@@ -108,19 +109,25 @@ public Summary describeDataRequestSummaryCases(String electionType) {
         if( !CollectionUtils.isEmpty(reviewedElections)){
             List<Integer> electionIds = reviewedElections.stream().map(e -> e.getElectionId()).collect(Collectors.toList());
             List<Vote> votes = voteDAO.findVotesByElectionIds(electionIds);
-            List<Vote> agreementVotes = votes.stream().filter(v -> v.getType().equals(VoteType.AGREEMENT.getValue())).collect(Collectors.toList());
-            if(CollectionUtils.isNotEmpty(agreementVotes)){
-                Map<Boolean, List<Vote>> partition =
-                        agreementVotes.stream()
-                                .collect(Collectors.partitioningBy(v -> v.getVote()));
-                summaryList.add(createSummary(0, partition.get(Boolean.TRUE).size(), partition.get(Boolean.FALSE).size()));
+
+            List<Vote> agreementYesVotes = filterAgreementVotes(votes, Boolean.TRUE);
+            List<Vote> agreementNoVotes = filterAgreementVotes(votes, Boolean.FALSE);
+
+            if (CollectionUtils.isNotEmpty(agreementYesVotes) || CollectionUtils.isNotEmpty(agreementNoVotes)) {
+                summaryList.add(createSummary(0, agreementYesVotes.size(), agreementNoVotes.size()));
             }
         }else{
             summaryList.add(createSummary(0,0,0));
         }
         return summaryList;
     }
 
+    private List<Vote> filterAgreementVotes(List<Vote> votes, Boolean desiredValue) {
+        return votes.stream().filter(
+                v -> v.getType().equals(VoteType.AGREEMENT.getValue()) && v.getVote() != null && v.getVote().equals(desiredValue)
```

Comment:
( this also isn't really important, don't hold the PR on this at all )

Reply:
It actually should support that case OK, because you can pass anything into a `.equals()`; I wanted to make the `null` check explicit since we (evidently) have not paid enough attention to `null` here in the past